src/data/thunks.js

```diff
@@ -272,6 +272,9 @@ export function pollAttempt(url) {
 
     try {
       const data = await pollExamAttempt(url);
+      if (!data) {
+        throw new Error('Poll Exam failed to fetch.');
```

Comment:
When for some reason the poll exam attempt fetch returns undefined, a really annoying error shows as:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'time_remaining_seconds').
```
Which is very unpractical. I'd rather have some better context if this happens.